ui/packages/catalog-ui-search/src/main/webapp/react-component/inputs/enum-input/enum-input.js

```diff
@@ -13,37 +13,89 @@
  *
  **/
 import React, { useState } from 'react'
+
 import Dropdown from '../../dropdown'
-import { Menu, MenuItem } from '../../menu'
+import { Menu, MenuItemDisabled } from '../../menu'
 import TextField from '../../text-field'
 import styled from 'styled-components'
 import { getFilteredSuggestions, inputMatchesSuggestions } from './enumHelper'
 import PropTypes from 'prop-types'
-
+const sources = require('../../../component/singletons/sources-instance')
 const TextWrapper = styled.div`
   padding: ${({ theme }) => theme.minimumSpacing};
 `
-
 const EnumMenuItem = props => (
-  <MenuItem {...props} style={{ paddingLeft: '1.5rem' }} />
+  <MenuItemDisabled {...props} style={{ paddingLeft: '1.5rem' }} />
 )
 
+const UnsupportedAttribute = styled.div`
+border-style: solid
+border-color: red
+`
+
+const isAttributeDisabled = (AllSupportedAttributes, currValue) => {
```

Comment:
Camel-cased identifiers that also start with a capital letter are typically used for class / type names
```suggestion
const isAttributeDisabled = (allSupportedAttributes, currValue) => {
```